debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -4800,6 +4800,7 @@ C|UTILIZATION_WINDOW_SIZE
 S|MmhookConstants|MmhookConstantsPointer|
 C|J9HOOK_MM_CLASS_UNLOADING_END
 C|J9HOOK_MM_INTERRUPT_COMPILATION
+C|J9HOOK_MM_WALKCONTINUATION
```

Comment:
I am not sure we can add new constants here. We need to double check

Reply:
Dmitri is correct: new constants cannot be added to this file.

Reply:
I also don't think it was necessary to add it.